python/ql/src/change-notes/2022-05-16-broken-crypto-block-mode.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+The query "Use of a broken or weak cryptographic algorithm" (`py/weak-cryptographic-algorithm`) now report if a cryptographic operation is potentially insecure due to use of a weak block mode.
```

Comment:
These should always start with `*` -- for example see [this fixup](https://github.com/github/codeql/pull/9141/commits/43265c41334f6af08dfcaa50d2062c0ba3b9356c)

```suggestion
* The query "Use of a broken or weak cryptographic algorithm" (`py/weak-cryptographic-algorithm`) now report if a cryptographic operation is potentially insecure due to use of a weak block mode.
```